specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -279,6 +279,223 @@ interface DevBoxes {
     }
   >;
 
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Need to add optional query param"
```

Comment:
Was this not possible by configuring a query parameter trait?

Reply:
https://azure.github.io/typespec-azure/docs/getstarted/azure-core/step09#queryparameterstraittparams-contexts